users/views.py

```diff
@@ -1,3 +1,78 @@
-from django.shortcuts import render
+import  json, re
+import  bcrypt
+import  jwt
+
+from    django.http                 import  JsonResponse
+from    django.views                import  View
+from    boomdoggy.settings          import  SECRET_KEY
+
+from    .models                     import  Users 
+
+
+
+class SignUp(View) : 
+    def post(self, request) : 
+        data = json.loads(request.body)
+
+        #비밀번호 암호화
+        hashed_password = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt())
+        
+        #이메일, 비밀번호 유효성
+        REGEX_EMAIL         = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+        REGEX_PASSWORD      = '^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,16}$'
+
+        try : 
+            if not (data['first_name'] and data['last_name']) : 
+                return JsonResponse({'messages' : 'PLEASE_INSERT_YOUR_NAME'}, status = 400)
+
+            if not re.match(REGEX_EMAIL, data['email']) : 
+                return JsonResponse({'message' : 'INVALID_ID'}, status = 401)
+
+            if not re.match(REGEX_PASSWORD, data['password']) : 
+                return JsonResponse({'message' : 'INVALID_PASSWORD'}, status = 401)
+
+            if Users.objects.filter(email = data['email']).exists() : 
+                return JsonResponse({'message' : 'ALREADY_EXISTS'}, status = 401)
+
+            else : 
+                Users.objects.create(
+                        first_name  =   data['first_name'],
+                        last_name   =   data['last_name'],
+                        email       =   data['email'],
+                        password    =   hashed_password.decode('utf-8'),
+                        )
+
+                return JsonResponse({'message' : 'SUCCESS'}, status = 201)
```

Comment:
```suggestion
            Users.objects.create(
                first_name  =   data['first_name'],
                last_name   =   data['last_name'],
                email       =   data['email'],
                password    =   hashed_password.decode('utf-8'),
            )

            return JsonResponse({'message' : 'SUCCESS'}, status = 201)
```
줄일 수 있는 else는 제거 해주세요!

Reply:
수정 반영 했습니다!